README.md

```diff
@@ -1,5 +1,31 @@
 # calens
 
-This program is a changelog generator that nicely prints a changelog file from
+`calens` is a changelog generator that nicely prints a changelog file from
 a directory with versioned subdirs containing files. A sample directory can be
 seen in [the restic repository](https://github.com/restic/restic/tree/master/changelog)
+
+# Installation
+
+To install and run calens locally, run the following commands:
+```
+git clone https://github.com/restic/calens.git 
+cd calens
+CGO_ENABLED=0 go install
+```
+
+Note that you need to to update the local repo and rerun installation when a new calens release becomes avialable.
```

Comment:
I think the upgrade instructions are redundant, as there's nothing special to do, just the standard set of updating the git repo and building the program again. Please remove that part again,